tests/Behat/Context/Api/ChannelPricingLogEntryContext.php

```diff
@@ -0,0 +1,105 @@
+<?php
```

Comment:
If we intend to integrate this plugin into core, what about sticking to the core paths?
In this case:
`tests/Behat/Context/Api/Admin/ChannelPricingLogEntryContext.php`